README.md

```diff
@@ -1731,6 +1732,13 @@ _See also: [awesome-sysadmin/Code Review](https://github.com/awesome-foss/awesom
 - [UVDesk](https://www.uvdesk.com/) - UVDesk community is a service oriented, event driven extensible opensource helpdesk system that can be used by your organization to provide efficient support to your clients effortlessly whichever way you imagine. ([Demo](https://demo.uvdesk.com/), [Source Code](https://github.com/uvdesk/community-skeleton)) `MIT` `PHP`
 - [ZenTao](https://www.zentao.pm/) - An agile(scrum) project management system/tool. ([Demo](https://demo.zentao.pm/user-login.html), [Source Code](https://github.com/easysoft/zentaopms)) `ZPL-1.2` `PHP`
 
+### Software Development - Security & QA
+
+**[`^        back to top        ^`](#)**
+
+- [Mega-Linter](https://megalinter.github.io/latest) - Mega-Linter can handle any type of project thanks to its 70+ embedded Linters, its advanced reporting, runnable on any CI system or locally, with assisted installation and configuration, able to apply formatting and fixes. `AGPL-3.0` `Python`
```

Comment:
> This is a list of Free Software [network services](https://en.wikipedia.org/wiki/Network_service) and [web applications](https://en.wikipedia.org/wiki/Web_application)

I don't think Mega-Linter is any of those, it looks like a command-line linter. You could submit it to https://github.com/ligurio/awesome-ci instead.

I think we should also add a link to https://github.com/ligurio/awesome-ci in https://github.com/awesome-selfhosted/awesome-selfhosted#software-development---continuous-integration--deployment

Reply:
@marcinguy ping, can you resolve the point mentioned above?

Reply:
@nodiscc Sorry did not see it.

Should I proceed with your modifications?

Reply:
@marcinguy yes please, let's remove Mega-Linter.